src/plugins/kibana/server/routes/api/ingest/register_processors.js

```diff
@@ -0,0 +1,24 @@
+import _ from 'lodash';
+import handleESError from '../../../lib/handle_es_error';
+import handleResponse from '../../../lib/process_es_ingest_processors_response';
+import { keysToCamelCaseShallow, keysToSnakeCaseShallow } from '../../../../common/lib/case_conversion';
+
+export function registerProcessors(server) {
+  server.route({
+    path: '/api/kibana/ingest/processors',
```

Comment:
Per our discussion, I agree that we can resolve this potential conflict by putting those urls under the `config` category.